oxidauth-usecases/src/users/create_user.rs

```diff
@@ -34,4 +34,7 @@ where
     ) -> Result<Self::Response, Self::Error> {
         self.users.call(req).await
     }
+
+    // if authority requires 2FA
```

Comment:
Would it be reasonable to create the user auth key regardless of whether or not the authority requires 2FA? If we do, then it is painless for an authority to switch to requiring 2FA at any time